QUICKSTART.md

```diff
@@ -0,0 +1,279 @@
+# **KubeStellar** Quickstart
+
+## Required Packages:
+   - [docker](https://docs.docker.com/get-docker/)
+   - [kind](https://kind.sigs.k8s.io/)
```

Comment:
The latest version of kind will install a too-recent version of Kubernetes, right?  We should say here which versions of kind are usable.

Reply:
Do you know which range of version for kind are usable?

Reply:
No

Reply:
I have not faced any issue with a version of kind yet - not sure which version range it would start to cause issues

Reply:
The latest version of kind will install Kubernetes release 1.26 or later.  kcp is based on Kubernetes release 1.24.  If I recall correctly, somebody reported a problem due to a difference between those two releases.

Reply:
for this release we are not going to be explicit about kind cluster versioning. If this becomes problematic, then we will address it in a PR for the next version of the quickstart. Just making an executive decision here so that we can converge and complete this PR.